src/components/semantic/presenters/questionPresenters.tsx

```diff
@@ -503,6 +503,7 @@ export function InlineQuestionPartPresenter(props: PresenterProps<IsaacInlinePar
             <NumericQuestionPresenter {...props} showMeta={false} />
         </>}
         {choices}
+        <AnswerPresenter {...props} />
         <SemanticDocProp {...props} prop="defaultFeedback" name="Default Feedback" />
```

Comment:
These two are the other way around (https://github.com/isaacphysics/isaac-content-editor/pull/74/files#diff-342ec15aaf1c437e651993e30f0c1ef83d294c7333ba9054c70447aee4236ba2R209) in ordinary questions; I think we should be consistent!

Reply:
```suggestion
        <SemanticDocProp {...props} prop="defaultFeedback" name="Default Feedback" />
        <AnswerPresenter {...props} />
```